src/stories/Table.story.vue

```diff
@@ -1,48 +1,49 @@
 <template>
   <Story title="Table">
     <Variant title="General">
-      <HoppSmartTable :list="list" :headings="headings" />
+      <HoppSmartTable
+        :headings="headings"
+        :list="list"
+        :checkbox="true"
+        :spinner="{ enabled: true }"
+        :selected-rows="selectedRows"
+        :pagination="{ totalPages: 2 }"
+        :search-bar="{ debounce: 1000, placeholder: 'Search by name' }"
+        :sort="{ key: 'name', direction: 'ascending' }"
```

Comment:
Can we also have a change handler for the search query filtering the results by name? If possible, let's also add a sort button that updates the list ordering.

Reply:
@JoelJacobStephen Search will be completely removed from the table. That will be handled by the parent component. And do we need a sort button in the table? I think that can be handled by the parent component where we can use an independent button and attach the sort direction to the direction variable under the sort prop.

Reply:
Yup, I was referring to demonstrating the associated behaviors via the story. In this case, adding a sort button aimed at toggling the value supplied to the `sort` prop for the `HoppSmartTable` component along with search.